dail/postponed-expedited-snap-verifications.vbs

```diff
@@ -104,13 +215,41 @@ DO
 		Dialog Dialog1
 		cancel_confirmation
 		Call validate_MAXIS_case_number(err_msg, "*")
-		IF verifs_needed = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
-		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
+		IF trim(verifs_needed) = "" THEN err_msg = err_msg & vbCr & "* Please enter the postponed verifications requested."
+		IF len(trim(verifs_needed)) > 100 THEN err_msg = err_msg & vbCr & "* The list of verifications must be less than 100 characters long. Please shorten."
+		IF trim(worker_signature) = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
 		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbNewLine & err_msg & vbNewLine		'error message including instruction on what needs to be fixed from each mandatory field if incorrect
 	LOOP UNTIL err_msg = ""									'loops until all errors are resolved
 	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
 LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 
+'Write information to SPEC/MEMO
+memo_header = "--- SNAP Closed " & closure_date & " - Expedited Autoclose ---"
+Call write_variable_in_SPEC_MEMO(memo_header)
+Call write_variable_in_SPEC_MEMO(" ")
+Call write_variable_in_SPEC_MEMO("Reason for closure: Delayed verifications were not submitted and Expedited SNAP Autoclosed.")
+Call write_variable_in_SPEC_MEMO(" ")
+If verifs_needed <> "" then call write_variable_in_SPEC_MEMO("Verifications needed: " & verifs_needed)
+Call write_variable_in_SPEC_MEMO(" ")
+CALL write_variable_in_SPEC_MEMO("*** Submitting Documents:")
```

Comment:
For lines 234-237, consider using: function digital_experience()
'--- This function standardizes the digital experience coding information so that we only have to update it in one location. 
'===== Keywords: MAXIS, MEMO, WCOM
	CALL write_variable_in_SPEC_MEMO("*** Options for Submitting Documents ***")
	CALL write_variable_in_SPEC_MEMO("- Use InfoKeep online at infokeep.hennepin.us to upload directly to your case.")
	CALL write_variable_in_SPEC_MEMO("- You can submit documents online at MNBenefits.mn.gov.")
	CALL write_variable_in_SPEC_MEMO("- Mail, Fax, or Drop Boxes at service centers.")
	Call write_variable_in_SPEC_MEMO(" ")
end function

Reply:
Great, thank you! I will utilize the function once I merge the other changes in and can utilize the new function from the functions library.

Reply:
I agree. I think as long as we have the new function in there, we don't need to include the list of service centers. We provide this information on other notices that are required, and this is very much a customer service type of notice.

Reply:
Great, makes sense. I will add the function once I merge the rest of the changes.